plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/NoopNameMangler.kt

```diff
@@ -10,8 +10,11 @@ import org.jetbrains.kotlin.fir.symbols.impl.FirValueParameterSymbol
 import org.jetbrains.kotlin.formver.embeddings.embedName
 
 object NoopNameMangler : NameMangler {
-    override fun mangleParameterName(parameter: FirValueParameterSymbol) = parameter.embedName()
-    override fun mangleLocalPropertyName(property: FirPropertySymbol) = property.callableId.embedName()
+    override fun mangleParameterName(parameter: FirValueParameterSymbol, scopeDepth: Int?) =
+        parameter.embedName(scopeDepth ?: throw IllegalArgumentException("null scope depth found while mangling $parameter"))
```

Comment:
Same here.  Is there any case when `scopeDepth` should return `null`?  It would be best to find the first point at which a null scope is not permitted and throw there.

Reply:
make it nullable was a patch for some issues related to inlining, I'll find a better solution :)